config/Config-images.in

```diff
@@ -310,7 +310,7 @@ menu "Target Images"
 		int "Kernel partition size (in MiB)"
 		depends on USES_BOOT_PART
 		default 8 if TARGET_apm821xx_sata
-		default 64 if TARGET_bcm27xx
+		default 64 if TARGET_bcm27xx || TARGET_x86_64 || TARGET_x86_generic
```

Comment:
Will sysupgrade work with this change?
I haven't fully understood what data will be lost when someone upgrades.

Reply:
I just built and booted into `r30381-b626262226a8` as-is, then built it again with this PR applied. I can confirm that updating with `sysupgrade` successfully rebooted without issue.

Reply:
The issue is with existing deployments. Sysupgrade compares present partition layout 16+104 with the one provided by new image, which, if differs, will overwrite whole disk. There's no issue if users use customized layout (1st+2nd partition), they're on their own anyway, but there's an issue if user uses default layout and creates 3rd+ partition. That partition will be wiped on sysupgrade if default layout of 1st+2nd partition will change[1].

So in all, sysupgrade will work regardless, the only issue mentioned was wiping user 3rd+ partition if we change sizes of 1st or 2nd partition. If we're fine with that caveat, there are no other issues.

1. https://github.com/openwrt/openwrt/blob/06267823a39c812e6ebea50264b0367ed8473ba4/target/linux/x86/base-files/lib/upgrade/platform.sh#L3

Reply:
Will the rootfs_data of a squashfs root file system be deleted?
Does this only affect when I add a custom partition directly into the 48 MB after the old rootfs?
Would I have the same problem when the rootfs increases by 48MB?

Reply:
There is a also a PR to increase the default rootfs size: #19388

Reply:
Is squashfs handled differently than ext4? I have no experience on x86/64 with squashfs/always have used the ext4 target.

Reply:
> There is a also a PR to increase the default rootfs size: https://github.com/openwrt/openwrt/pull/19388

I like the idea of that PR getting merged before this one